### PR TITLE
feat(components/atom/tag): Customize background for close icon

### PR DESCRIPTION
**Pull request by manjarcode**
Allows to customize the close icon background and color when mouse hover

## Category/Component

`❓ Ask`

**TASK**:  https://jira.scmspain.com/browse/PI-52232

### Types of changes

- [ ] 🪲 Bug fix (non-breaking change which fixes an issue)
- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🧾 Documentation
- [ ] 📷 Demo
- [ ] 🧪 Test
- [ ] 🧠 Refactor
- [ ] 💄 Styles

### Description, Motivation and Context
In jobs we want to customize the close icon hover color to improve the UX
https://www.figma.com/file/x80a9OA3Q6bk182smlfZsl/CV_Refactor?node-id=2002%3A334&t=RsscI4T8ihl0YsBp-0
### Screenshots - Animations
We want to customize this default close icon color
image

According to this we want to customize this default close icon background color (when hover)
image

The custom type config for the icon color
image

The custom type config for the icon background (when hover)
image

### CODE REVIEW

**Review by andresin87 — APPROVED** [review]

File: components/atom/tag/src/styles/index.scss [hunk 1]
```diff
@@ -206,6 +209,7 @@ $base-class: '.sui-AtomTag';
       fill: $c;
 
       &#{$self}-actionable {
+        border: 1px solid;
```
Comment (beatrizip) [inline components/atom/tag/src/styles/index.scss]:
```suggestion
        border: $bdw-s solid;
```

[SUI-Theme](https://github.com/SUI-Components/sui/blob/master/packages/sui-theme/src/settings/_box-style.scss#L25)

File: components/atom/tag/src/styles/index.scss [hunk 2]
```diff
@@ -219,6 +223,18 @@ $base-class: '.sui-AtomTag';
         fill: $bgc;
       }
 
+      & > #{$self}-closeable {
```
Comment (beatrizip) [inline components/atom/tag/src/styles/index.scss:225]:
```suggestion
      & > #{$self}-action {
```

File: components/atom/tag/src/styles/index.scss [hunk 3]
```diff
@@ -219,6 +223,18 @@ $base-class: '.sui-AtomTag';
...
+        color: $c-icon-close;
+      }
+
+      #{$self}-closeable > #{$self}-closeableIcon {
```
Comment (beatrizip) [inline components/atom/tag/src/styles/index.scss:229]:
```suggestion
      #{$self}-action > #{$self}-actionIcon {
```

File: components/atom/tag/src/styles/index.scss [hunk 4]
```diff
@@ -219,6 +223,18 @@ $base-class: '.sui-AtomTag';
...
+        color: $c-icon-close;
```
Comment (beatrizip) [inline components/atom/tag/src/styles/index.scss:230]:
```suggestion
        color: $c-icon-action;
```

File: components/atom/tag/src/styles/index.scss (hunk 1 above)
Comment (manjarcode) [reply components/atom/tag/src/styles/index.scss]: I missed this one! it was only for testing I will remove it!

File: components/atom/tag/src/styles/index.scss [hunk 5]
```diff
@@ -219,6 +223,18 @@ $base-class: '.sui-AtomTag';
...
+        &:hover {
+          color: $c-icon-close-hover;
```
Comment (beatrizip) [inline components/atom/tag/src/styles/index.scss:232]:
```suggestion
          color: $c-icon-action-hover;
```

File: components/atom/tag/src/styles/index.scss [hunk 6]
```diff
@@ -219,6 +223,18 @@ $base-class: '.sui-AtomTag';
...
+          background-color: $bgc-icon-close-hover;
```
Comment (beatrizip) [inline components/atom/tag/src/styles/index.scss:233]:
```suggestion
          background-color: $bgc-icon-action-hover;
```

File: components/atom/tag/src/styles/index.scss (hunk 2 above)
Comment (manjarcode) [reply components/atom/tag/src/styles/index.scss:225]: This one refers to the class sui-AtomTag-closeable I can't change the name

File: components/atom/tag/src/styles/index.scss (hunk 3 above)
Comment (manjarcode) [reply components/atom/tag/src/styles/index.scss:229]: same here the class is sui-AtomTag-closeableIcon

File: components/atom/tag/src/styles/index.scss (hunk 2 above)
Comment (manjarcode) [reply components/atom/tag/src/styles/index.scss:225]: https://github.com/SUI-Components/sui-components/blob/b22b29db13ed763aeb08f9d39ad24acf3bf0309c/components/atom/tag/src/Standard.js#L30

File: components/atom/tag/src/styles/index.scss (hunk 3 above)
Comment (manjarcode) [reply components/atom/tag/src/styles/index.scss:229]: https://github.com/SUI-Components/sui-components/blob/b22b29db13ed763aeb08f9d39ad24acf3bf0309c/components/atom/tag/src/Standard.js#L36

**Review by jcger — APPROVED** [review]

**Review by beatrizip — APPROVED** [review]

**Pull request merged**